public/js/index.js

```diff
@@ -1,57 +0,0 @@
-(()=>{
-  dom();
-})();
```

Comment:
What I mean is, why define the `dom` function, why not just wrap an anonymous function? You could also:
```
(() => {
  // dom code
})();
```